src/node_modules/util/string.mjs

```diff
@@ -0,0 +1,13 @@
+import * as _ from 'lamb'
+import { trim } from '@svizzle/utils'
+
+
+export const toLowerString = v => v.toString().toLowerCase()
+// tag function to dedent template literals
+export const dedent = _.pipe([
```

Comment:
formatting: I'd have a single blank line between each function and between the imports block and the first function